q2_longitudinal/_utilities.py

```diff
@@ -518,6 +518,36 @@ def _visualize(output_dir, multiple_group_test=False, pairwise_tests=False,
     })
 
 
+def _visualize_anova(output_dir, pairwise_tests=False, model_results=False,
+                     residuals=False, pairwise_test_name='Pairwise t-tests'):
+    pd.set_option('display.max_colwidth', -1)
+
+    if pairwise_tests is not False:
+        pairwise_tests.to_csv(os.path.join(output_dir, 'pairwise_tests.tsv'),
+                              sep='\t')
+        pairwise_tests = q2templates.df_to_html(pairwise_tests)
+
+    if model_results is not False:
+        model_results.to_csv(os.path.join(output_dir, 'model_results.tsv'),
+                             sep='\t')
+        model_results = q2templates.df_to_html(model_results)
+
+    if residuals is not False:
```

Comment:
Can you add a unit test that looks at the presence of these files when this test is performed?

Reply:
never false, so have removed if statement